bin/bazelify.dart

```diff
@@ -23,7 +22,16 @@ Future<Null> main(List<String> args) async {
     return;
   }
 
-  await Chain.capture(() => generate(arguments), onError: (error, chain) {
+  await Chain.capture(() async {
+    if (arguments is BazelifyInitArguments) {
+      await generate(arguments);
+    } else if (arguments is BazelifyServeArguments) {
+      await serve(arguments);
+    } else {
+      throw new StateError('Something has gone horribly wrong, please file a '
```

Comment:
I think we want print + set exit code. Users don't want to see a stack trace when this happens